x-pack/plugins/apm/server/routes/services/get_services/get_service_alerts.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  AggregationsCardinalityAggregate,
+  AggregationsFilterAggregate,
+} from '@elastic/elasticsearch/lib/api/types';
+import { kqlQuery, rangeQuery } from '@kbn/observability-plugin/server';
+import { ALERT_RULE_PRODUCER, ALERT_STATUS } from '@kbn/rule-data-utils';
+import { SERVICE_NAME } from '../../../../common/es_fields/apm';
+import { ServiceGroup } from '../../../../common/service_groups';
+import { serviceGroupQuery } from '../../../lib/service_group_query';
+import { ApmAlertsClient } from '../../../lib/helpers/get_apm_alerts_client';
+
+interface ServiceAggResponse {
+  buckets: Array<
+    AggregationsFilterAggregate & {
+      key: string;
+      alerts_count: AggregationsCardinalityAggregate;
+    }
+  >;
+}
+
+export async function getServicesAlerts({
+  apmAlertsClient,
+  kuery,
+  maxNumServices,
+  start,
+  end,
+  serviceGroup,
+}: {
+  apmAlertsClient: ApmAlertsClient;
+  kuery: string;
+  maxNumServices: number;
+  start: number;
+  end: number;
+  serviceGroup: ServiceGroup | null;
+}) {
+  const params = {
+    size: 0,
+    query: {
+      bool: {
+        filter: [
+          { term: { [ALERT_RULE_PRODUCER]: 'apm' } },
+          { term: { [ALERT_STATUS]: 'active' } },
+          ...rangeQuery(start, end),
```

Comment:
```suggestion

```

We don't apply a time range on the service groups page. Should we then do it for inventory?

https://github.com/elastic/kibana/blob/a9166da678e34777e0cea03f9d667441a1a16d66/x-pack/plugins/apm/server/routes/service_groups/get_service_group_alerts.ts#L43-L66

Reply:
We might need to give more thinking to changing the timeframe because if we change it through the query params this will stay for other pages as well. I think this could impact performance.